managers/paranet-operations-manager.js

```diff
@@ -0,0 +1,384 @@
+const {resolveUAL, getOperationStatusObject} = require('../services/utilities.js');
+const { ethers } = require('ethers');
+
+class ParanetOperationsManager {
+    constructor(services) {
+        this.blockchainService = services.blockchainService;
+        this.inputService = services.inputService;
+        this.nodeApiService = services.nodeApiService;
+        this.validationService = services.validationService;
+    }
+
+    async create(UAL, options = {}) {
+        const {
+            blockchain,
+            paranetName,
+            paranetDescription,
+        } = this.inputService.getParanetCreateArguments(options);
+
+        this.validationService.validateParanetCreate(
+            UAL,
+            blockchain,
+            paranetName,
+            paranetDescription,
+        );
+
+        const { contract, tokenId } = resolveUAL(UAL);
+
+       await this.blockchainService.registerParanet({
+                contract,
+                tokenId,
+                paranetName,
+                paranetDescription,
+            },
+            blockchain
+        );
+
+        return {
+            paranetUAL: UAL,
+        };
+    }
+
+    async deployIncentivesContract(paranetUAL, incentiveType, options = {}) {
+        const {
+            blockchain,
+            tracToNeuroEmissionMultiplier,
+            operatorRewardPercentage,
+            incentivizationProposalVotersRewardPercentage,
+        } = this.inputService.getParanetDeployIncentivesContractArguments(options);
+
+        this.validationService.validateDeployIncentivesContract(
+            paranetUAL,
+            blockchain,
+            tracToNeuroEmissionMultiplier,
+            operatorRewardPercentage,
+            incentivizationProposalVotersRewardPercentage,
+        );
+        if(incentiveType === 'Neuroweb') {
```

Comment:
Can we crate some constanst in constasts which will emulate enum like INCENTIVE_TYPE.NEUROWEB = 1and INCENTIVE_TYPE.NOT_NEUROWEB = 0

Reply:
Why would we assign numbers and have not neuroweb?

Reply:
Thats also good, just change it to be constant